readthedocs/gold/templates/gold/subscription_form.html

```diff
@@ -43,12 +43,40 @@
       product.
     {% endblocktrans %}
   </p>
+    <p>
+    {% blocktrans %}
+      We ask that companies that are using Read the Docs support with at least $20/mo to help support the project.
+      If you are an individual,
+      feel free to give whatever feels right for the value you get out of Read the Docs.
```

Comment:
I think we should mention individuals first, as that is supposed to be the primary user of the community site. For companies, this does suggest that gold subscriptions covers our costs from companies, so maybe we need more here. I'd rather have something, rather than nothing from companies, but I imagine companies would rather pay for support than they would hand out money for support. Maybe we either push companies towards support contracts/hosting plans, rather than ask for money here, or we ask in addition -- ie, not this but something like it: "consider a support contract if support is important to your company, or if your company finds read the docs helpful, consider at least donating"